### ISSUE TITLE
(a11y) -- menu is broken at higher zoom levels

### ISSUE THREAD

**Issue by ctcpip** [expressjs/expressjs.com#1731]
zoom to 400%

only some menu items appear, and users cannot scroll to see all the menu items

![Screenshot 2025-01-13 at 2 53 07 PM](https://github.com/user-attachments/assets/3ac89de9-ae02-4c45-b9b4-7bb00c1ff35f)

**Comment by bjohansebas** [comment]
Hopefully, one day we can migrate the website and not have to worry about this 😢

**Comment by ctcpip** [comment]
until then, it would be great if we could help our users with low vision to be able to use the site  😊

**Comment by UndefinedYara** [comment]
@ctcpip @bjohansebas Good day guys! can this be assigned to me? I can help!

**Comment by ShubhamOulkar** [comment]
@UndefinedYara, you can work on it. We don't assign task.
